.travis.yml

```diff
@@ -1,25 +1,31 @@
 language: php
+sudo: false
 
 php:
+    - 7.1
```

Comment:
I would quote the version. GitHub/Travis had some issues with their parser at some point and where wrongly converting the version which should be a `string` into a `int` or `float` causing some issues (notably with nodejs builds). Since then they recommend explicitly putting strings. 

I also think we should add a PHP nightly build as we have an HHVM one.